File: Architecture-JEE-Middlewares/TP/Digital Banking/bank-Frontend-web/src/app/Model/account.model.ts
export interface AccountDetails {
  accountId:            string;
  balance:              number;
  currentPage:          number;
  totalPages:           number;
  pageSize:             number;
  accountOperationDtos: AccountOperationDto[];
}

export interface AccountOperationDto {
  id:            number;
  operationDate: Date;
  amount:        number;
  description:   string;
  type:          string;
}

File: Architecture-JEE-Middlewares/TP/Digital Banking/front_mobile/lib/Model/account.model.dart
class AccountDetails {
          String accountId ;
            double balance ;
         int currentPage ;
           int totalPages;
          int pageSize  ;
List< AccountOperationDto> accountOperationDtos=[];
AccountDetails(this.accountId,this.balance,this.currentPage,this.totalPages,this.pageSize);
}

class AccountOperationDto {
 int id;
DateTime operationDate;
double amount;
 String description;
String type;

AccountOperationDto(this.id,this.operationDate,this.amount,this.description,this.type);
}